backend/habitat/viewsets.py

```diff
@@ -870,3 +1083,82 @@ def bathymetry_statistics(request):
             return Response([])
         else:
             return Response(bathymetry_stats)
+
+@action(detail=False)
+@api_view()
+@renderer_classes((JSONRenderer, ShapefileRenderer))
```

Comment:
It's not a bad idea, but is there a concrete reason for ShapefileRenderer to be included here?  (more specifically, have you tested it?)

Reply:
No reason, just a copy+paste error